src/EnergyPlus/WindowManager.hh

```diff
@@ -153,6 +163,43 @@ namespace WindowManager {
 
     //****************************************************************************
 
```

Comment:
The code chunk computing the coefficient matrix of the heat balance equations is separated out from the function WindowTempsForNominalCond.